stock_barcodes/tests/test_stock_barcodes_inventory.py

```diff
@@ -0,0 +1,178 @@
+from odoo.tests.common import tagged
+
+from .test_stock_barcodes import TestStockBarcodes
+
+
+@tagged("post_install", "-at_install")
+class TestStockBarcodesInventory(TestStockBarcodes):
```

Comment:
Please add docstring and assertion message.

Reply:
done!